addons/mail/static/src/components/attachment/attachment.xml

```diff
@@ -10,18 +10,20 @@
                 'o-temporary': attachment and attachment.isTemporary,
                 'o-viewable': attachment and attachment.isViewable,
             }" t-att-title="attachment ? attachment.displayName : undefined" t-att-data-attachment-local-id="attachment ? attachment.localId : undefined"
+                role="menu" t-att-aria-label="attachment and attachment.displayName"
         >
             <t t-if="attachment">
                 <!-- Image style-->
                 <!-- o_image from mimetype.scss -->
-                <div class="o_Attachment_image o_image" t-on-click="_onClickImage"
+                <a class="o_Attachment_image o_image" t-on-click="_onClickImage"
```

Comment:
don't forget empty href

Reply:
Not necessary, this tag already has a valid href

Reply:
`div` had `href`, interesting... obviously ok then, comment can be ingored